src/main.js

```diff
@@ -171,6 +177,14 @@ import AppMain from './components/AppMain'
     Vue.prototype.$window = window
     Vue.prototype.OC = OC
 
+    //Markdown for Vue
+    Vue.use(VueShowdown, {
+        //set default flavor for Markdown
+        flavor: 'github',
```

Comment:
If we see the use case that our MD content is used in the other apps, I would suggest to use the `vanilla` flavor. [It seems]() that the NC Notes devs are going to support only a very basic set of Markdown features. 

If it’s only the other way round (copying from NC Notes to Cookbook), I would vote for a more feature-rich flavor.

Reply:
Independently of other NC Apps: The supported flavor of the editor and viewer component should probably be the same